models/scripts/SentimentAnalysis.py

```diff
@@ -1,6 +1,9 @@
 from tabpy_tools.client import Client
 from textblob import TextBlob
-from nltk.sentiment import SentimentIntensityAnalyzer
+import nltk
+nltk.download('vader_lexicon')
```

Comment:
Why are these downloads necessary? Does pip installing nltk not install everything that's needed?

Reply:
didn't work for me without that

Reply:
Yes the download is necessary. nltk does not install the vader lexicon by defaul, which is required for sentiment analysis in nltk. It's essentially a text file that contains the known sentiment scores of various words.